pkg/agent/openflow/pipeline.go

```diff
@@ -288,6 +294,34 @@ func GetAntreaPolicyEgressTables() []*Table {
 	}
 }
 
+func GetAntreaIGMPTables() []*Table {
+	return []*Table{
+		MulticastIGMPEgressTable,
+		MulticastIGMPIngressTable,
+		MulticastIGMPIngressMetricTable,
```

Comment:
Would you add comments to explain why we don't have MulticastEgressMetricTable?

Reply:
done, added at definition.